graylog2-web-interface/src/components/rules/RuleSimulation.tsx

```diff
@@ -40,33 +41,40 @@ const MessageShowContainer = styled.div`
   padding: 16px;
 `;
 
+const ActionOutputIndex = styled.b`
+  color: #aaa;
+`;
+
+const OutputContainer = styled.div`
+  margin-top: 8px;
+`;
+
 type Props = {
   rule?: RuleType | RuleBuilderRule,
 };
 
 const RuleSimulation = ({ rule: currentRule }: Props) => {
```

Comment:
When I copy a message and use it as the value in the Rule simulation. The simulation result reset with the default simulation message. See below.
![rule-simulation](https://github.com/Graylog2/graylog2-server/assets/1381455/1a855788-3dac-47a3-894a-0fa8f2962583)